packages/qwik/src/optimizer/core/src/snapshots/qwik_core__test__example_jsx.snap

```diff
@@ -58,17 +60,15 @@ import { Fragment as _Fragment } from "@qwik.dev/core/jsx-runtime";
 export const Lightweight = (props)=>{
     return /*#__PURE__*/ _jsxSorted("div", null, null, /*#__PURE__*/ _jsxSorted(_Fragment, null, null, [
         /*#__PURE__*/ _jsxSorted("div", null, null, null, 3, null),
-        /*#__PURE__*/ _jsxSplit("button", {
-            ...props
-        }, null, null, 0, null)
+        /*#__PURE__*/ _jsxSplit("button", _getVarProps(props), _getConstProps(props), null, 0, null)
```

Comment:
so, shouldn't this be `{..._getVarProps(props)}`? Or maybe _getVarProps should get a flag to return a shallowly cloned object?
Or we disallow shallowly changing the props object, as an optimization?